styles.css

```diff
@@ -316,7 +339,43 @@ ul > li {
 .user-tool-d {
   display: flex;
   align-items: center;
-  justify-content: space-between;
+}
+
+.user-tool-d img {
+  padding-left: 50%;
+ 
+}
+
+.skills img {
+  padding-left: 10%;
+}
+
+.languages {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: 500;
+}
+
+.skills {
+  display: flex;
+  align-items: center;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: 500;
+}
+
+.frameworks {
+  display: flex;
```

Comment:
- [ ] As requested by the previous code reviewer, you should make sure to bring the directional arrow to the right side of each of the Languages, Frameworks, and Skills that is missing in your design as you can see in the Figma Design.

- [ ] Also while working on the changes the previous code reviewer requested, your languages icons are not showing as it is in the Figma design.
  
  ### YOUR DESIGN
  ![image](https://user-images.githubusercontent.com/92303891/221381415-731617dc-dac0-4264-b03b-ba990e9706a1.png)
  
  ### FIGMA DESIGN
![image](https://user-images.githubusercontent.com/92303891/221381511-a82612fc-5f48-4234-b768-8324f3c732b7.png)